test-apps/ui-test-app/src/frontend/appui/frontstages/component-examples/ComponentExamplesProvider.tsx

```diff
@@ -1,1379 +1,1379 @@
-/*---------------------------------------------------------------------------------------------
-* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
-* See LICENSE.md in the project root for license terms and full copyright notice.
-*--------------------------------------------------------------------------------------------*/
-/* eslint-disable deprecation/deprecation */

```

Comment:
Can you flip the line endings here?

Reply:
Is there any way to tell here via github if it is CRLF or LF endings? Because I'm getting inconsistent results locally in VSCode. Whenever I try to convert to force the file to LF there is nothing to stage but when I force to CRLF it lets me stage the file. So maybe it was previously CRLF and it this change is to LF?